docs/docs-content/clusters/edge/edgeforge-workflow/palette-canvos/signed-content.md

```diff
@@ -0,0 +1,166 @@
+---
+sidebar_label: "Embed a Public Key in Edge Artifacts"
+title: "Embed a Public Key in Edge Artifacts"
+description: "Learn how to embed a public key in Edge artifacts to ensure the authenticity of all uploaded content."
+icon: ""
+hide_table_of_contents: false
+sidebar_position: 50
+tags: ["edge"]
+---
+
+EdgeForge gives you the option to embed public keys in the installer ISO and provider images. When an Edge host has an
+embedded public keys, all content uploaded to the Edge host, including content bundles and cluster definitions must
+contain a signature from the corresponding private key. This allows you to be confident that all content uploaded to
+your Edge hosts come from a trusted source. For more information about content bundles and cluster definitions, refer to
+[Build Content Bundle](./build-content-bundle.md) and
+[Export Cluster Definition](../../local-ui/cluster-management/export-cluster-definition.md).
+
+## Limitations
+
+- This feature is only available to Edge hosts without a connection to Palette.
+
+## Prerequisites
+
+- Embedding public keys is part of the EdgeForge process. Become familiar with EdgeForge before proceeding with this
+  guide. For more information, refer to [EdgeForge](../edgeforge-workflow.md).
+
+- [Git](https://git-scm.com/downloads). You can ensure git installation by issuing the `git --version` command.
+
+- [openssl](https://www.openssl.org/docs). You can ensure openssl installation by issuing the `openssl --help` command.
+
+- An RSA key pair. Both the public and private key must be in PEM format. You can use `openssl` to convert keys from
+  other formats into PEM. For more information, refer to
+  [openssl documentation](https://www.openssl.org/docs/manmaster/man1/openssl.html).
+
+- A physical or virtual Linux machine with _AMD64_ (also known as _x86_64_) processor architecture to build the Edge
+  artifacts. You can issue the following command in the terminal to check your processor architecture.
+
+  ```bash
+  uname -m
+  ```
+
+- Minimum hardware configuration of the Linux machine:
+
+  - 4 CPU
+  - 8 GB memory
+  - 150 GB storage. If you plan on using a content bundle, the actual storage will depend on the size of the content
+    bundle you will use to build the Edge installer ISO image.
+
+## Embed a Public Key in Edge Artifacts
+
+1. Clone the **CanvOS** repository.
+
+   ```shell
+    git clone https://github.com/spectrocloud/CanvOS.git
+   ```
+
+2. Change to the **CanvOS/** directory.
+
+   ```shell
+   cd CanvOS
+   ```
+
+3. View the available git tag.
+
+   ```shell
+   git tag
+   ```
+
+4. Check out the latest available tag. This guide uses the tag v4.4.a as an example.
+
+   ```
+   git checkout v4.4.a
+   ```
```

Comment:
This will not be the final version - so we need to remember and fix this before the release.